code/_onclick/hud/parallax.dm

```diff
@@ -310,7 +311,7 @@ INITIALIZE_IMMEDIATE(/atom/movable/screen/parallax_layer)
 
 /atom/movable/screen/parallax_layer/random
 	blend_mode = BLEND_OVERLAY
-	speed = 3
+	speed = 1
```

Comment:
This should probably be changed back now if the nebula's going onto its own 'layer':
```suggestion
	speed = 3
```